Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityControllerConfigurationProfile.asset

```diff
@@ -1120,3 +1120,157 @@ MonoBehaviour:
       axisCodeX: 
       axisCodeY: 
       invertYAxis: 0
+  - id: 8
+    description: WMR Source Unknown Handedness
```

Comment:
Windows Mixed Reality Hand Gestures?

Reply:
I left this more generic, in case there's an unhanded WMR controller out there. We're focusing more on hands in this implementation (and I basically only filled in `select` anyway haha...), so I'll update the name to your suggestion.